hybrasyl/Utility.cs

```diff
@@ -280,6 +281,116 @@ static class LogLevels
         public const int DEBUG = 7;
     }
 
+    class GameFolders
+    {
+        private static GameFolders instance;
+
+        private GameFolders() {}
+
+        public static GameFolders Instance
+        {
+            get
+            {
+                if (instance == null)
+                {
+                    instance = new GameFolders();
+                }
+                return instance;
+            }
+        }
+        //Adjusted for ease of future changes, if needed
+        private string _dataDirectory => Constants.DataDirectory;
+        private string _worldFileDirectory => Path.Combine(_dataDirectory, "world");
+        private string _mapFileDirectory => Path.Combine(_worldFileDirectory, "mapfiles");
+        private string _scriptsFileDirectory => Path.Combine(_worldFileDirectory, "scripts");
+        private string _xmlFileDirectory => Path.Combine(_worldFileDirectory, "xml");
+
+        public string MapFileDirectory => Path.Combine(_mapFileDirectory);
+
+        #region ScriptsFolders
+        public string ScriptCastableDirectory => Path.Combine(_scriptsFileDirectory, "castable");

```

Comment:
Same here, these should all be `public static string`.

Reply:
Fixed